### PR TITLE
add docs link

### PR DESCRIPTION
**Pull request by Cyberface**

### CODE REVIEW

**Review by vivienr — DISMISSED** [review]
LGTM

File: README.rst [hunk 1]
```diff
@@ -11,6 +11,14 @@ Installation
 
    pip install gwin
 
+
+Documentation
+=============
+
+- http://gwin.readthedocs.io/
```
Comment (duncanmmacleod) [inline README.rst]: @Cyberface, can you use the https URL instead?

**Review by duncanmmacleod — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (Cyberface) [reply README.rst]: I don't know what you mean :'(

Comment (Cyberface) [reply README.rst]: OH ! I get you the 's' :)

**Review by duncanmmacleod — APPROVED** [review]

**Pull request merged**